catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactory.java

```diff
@@ -133,7 +137,12 @@ public QueryRequest getQuery(GetRecordsType request) throws CswException {
 
         CswRecordMapperFilterVisitor filterVisitor = buildFilter(query.getConstraint());
         QueryImpl frameworkQuery = new QueryImpl(filterVisitor.getVisitedFilter());
-        frameworkQuery.setSortBy(buildSort(query.getSortBy()));
+        SortBy[] sortBys = buildSort(query.getSortBy());
+        SortBy[] extSortBys = null;
+        if (sortBys != null && sortBys.length > 0) {
+            frameworkQuery.setSortBy(sortBys[0]);
+            extSortBys = (Arrays.stream(sortBys).skip(1).collect(Collectors.toList())).toArray(new SortBy[0]);
```

Comment:
wouldn't it be easier to use something like `extSortBys = Arrays.copyOfRange(sortBys, 1, sortBys.length);`?